### ISSUE TITLE
Motion sensor doesn't work on v1.5.17

### ISSUE THREAD

**Issue by alpol017** [athombv/homey#2445]
**Metadata**

* **Homey Version**:
* **Homey Log Code**:
* **Smartphone Platform**:
* **Smartphone App Version**:

**If related to a Homey App:**
* **Homey App Version**:
* **Homey App Log Code**:
* **Affected Devices (preferably product codes)**:

**What did you try to achieve?**

**What did you expect as result?**

**What was the result?**

**Any other remarks**

**Comment by alpol017** [comment]
Hello! Sorry for bad description.
I wrote about it on https://github.com/athombv/com.fibaro/issues/208

After update on latest fibaro app and Athom homey app, my two fibaro multisensors are not working correctly.

For example, one sensor detects motion and that’s all, it will be YES motion forever. It doesn’t get cancel alarm message.

I have reset my z-wave network, reinstall fibaro app and the same issue.

All the other devices (neo, aeotec) are working. NON-battery fibaro devices also works. 

If I put fibaros multisensors near homey they are working. If I move them for a 2-3 meters they are don’t, BUT I have seen in dev.tools-z-wave network, that homey receive messages from sensors but there is nothing in homey app.

BUT! Yesterday one of the sensors started to work as it should. I have placed it in another room and it’s working! But the second one doesn’t.

Alexandr Polovinko
